src/ndpp_init.F90

```diff
@@ -0,0 +1,604 @@
+module ndpp_initialize
+
+  use ace_header,   only: Nuclide, XsListing
+  use constants
+  use dict_header,  only: DictCharInt, DICT_NULL
+  use error,        only: fatal_error, warning
+  use global
+  use ndpp_header,  only: Ndpp
+  use ndpp_ops,     only: ndpp_read
+  use output,       only: write_message
+  use search
+  use string,       only: ends_with, to_lower, starts_with, to_str
+  use xml_interface
+
+  implicit none
+
+  ! ndpp_lib.xml preprocessed data listings and associated data.
+  type(XsListing), allocatable, target, save :: ndpp_listings(:)
+  type(DictCharInt), save                    :: ndpp_listing_dict
```

Comment:
`SAVE` is implicit for module variables. From the F2008 standard:

*A variable, common block, or procedure pointer declared in the scoping unit of a main program, module, or submodule implicitly has the SAVE attribute, which may be confirmed by explicit specification.*